app/models/flight.rb

```diff
@@ -4,10 +4,6 @@ class Flight < ActiveRecord::Base
   has_many :bookings
   has_many :passengers
 
-  before_save do
-    self.price = Faker::Commerce.price
-  end
-
   scope :location, (lambda do |loc|
     where("departure_location_id = ?", loc)
   end)
```

Comment:
**Usage of scopes**

I decided to leave the `filter` method as is because I found it still necessary since it's a utility method used to chain the scopes `#when`, `#location` & `#destination`.